### ISSUE TITLE
Unsuspend linux host

### ISSUE THREAD

**Issue by stuaxo** [utmapp/UTM#5636]
Host:  M2 Mac
Guest:  Qemu - Aarch64 (Arm64) ubuntu

On my real linux laptop, I can suspend it and then unsuspend by hitting a key or opening the lid.

On here, if I run `systemctl suspend` then try and press keys or move the mouse in the window, the guests screen remains black.

**Comment by osy** [comment]
I've briefly looked into this but I can't figure out how to resume it from QEMU. If you can figure out how to get it working on QEMU, we can add it.

**Comment by stuaxo** [comment]
The savevm/loadvm monitor commands look useful:
https://superuser.com/a/1785127

(scroll down "To create a snapshot in an online VM")

**Comment by osy** [comment]
@stuaxo we already do that when you press the "pause" button. I think you're asking for something different which is: if a guest decides to go to sleep, QEMU should be aware and not freeze up. If you can get that to work in a QEMU environment, let me know what commands you've used.

**Comment by stuaxo** [comment]
The pause button is always greyed out for me, do I need to change something to enable it ?

**Comment by osy** [comment]
Yes, that means that "savevm"/"restorevm" is not supported for your configuration. If you run the same VM in QEMU and try to run "savevm" it will give you an error.

**Comment by stuaxo** [comment]
You're right on the first point - I have derailed this, in that this issue is about the host being able to restore a suspended guest.

I'll try and work out how to use the qemu monitor and work out what's up.

**Comment by stuaxo** [comment]
Closing, since I no longer use a Mac with UTM+Qemu so won't be able to reproduce this or give further feedback.

**Comment by volkert-fastned** [comment]
I ran into a similar issue while running An Alpine Linux (with GNOME+Wayland desktop) under UTM 4.5.3. Aarch64 architecture, on an Apple Silicon host. As a workaround, I switched off auto-suspend in the GNOME settings, but it would be nice to have suspend/resume work properly in Linux VM guests like this.
